pnpm-workspace.yaml

```diff
@@ -5,3 +5,4 @@ packages:
   - 'website'
   - 'stories'
   - vite-project
+  - 'stories'
```

Comment:
we've already have stories on the 6th line

Reply:
Fixed